package/kernel/linux/modules/netdevices.mk

```diff
@@ -1219,3 +1219,18 @@ define KernelPackage/sfp/description
 endef
 
 $(eval $(call KernelPackage,sfp))
+
+define KernelPackage/igc
+  SUBMENU:=$(NETWORK_DEVICES_MENU)
+  TITLE:=Intel(R) Ethernet Controller I225 Series support
+  DEPENDS:=@PCI_SUPPORT @LINUX_5_4
```

Comment:
Please use @!LINUX_4_19 this way it wil automatically work with more recent kernel versions.

Reply:
done